### DISCUSSION TITLE
update all projects fails

### DISCUSSION THREAD

**Discussion by reginaldStjohn** [bitcraze/discussions#545] [General]
I just ran the BitcrazeVM in VirtualBox. Following the instructions from[https://www.bitcraze.io/documentation/tutorials/getting-started-with-crazyflie-2-x/#inst-virtualmachine]. The update fails with several errors.

![image](https://user-images.githubusercontent.com/117384888/216187462-b7c23589-f841-428a-8d73-57cb856201ed.png)

**Comment by tobbeanton** [comment]
upgrading pip solves this issue:
`pip3 install --upgrade pip`

**Reply by eunseo01** [reply]
Hi,
I also saw the same error. I entered the code you provided in your answer and got the same result as the first picture.
![image](https://user-images.githubusercontent.com/107001681/220345126-0d04983d-ab83-41da-baa3-07d4eecd67f7.png)

Then, when I tried updating again, I saw an error related to cflib's setup.py as shown in the attached picture. how can i solve this?
![image](https://user-images.githubusercontent.com/107001681/220340438-4bc0428a-9d50-40ae-9afb-5965f5d90fb9.png)
![image](https://user-images.githubusercontent.com/107001681/220340479-f67111ac-b6d2-4361-978b-c622096fb694.png)
![image](https://user-images.githubusercontent.com/107001681/220340511-da00409c-30cf-4024-82fd-3a16949ae397.png)
![image](https://user-images.githubusercontent.com/107001681/220340578-2836d976-b123-4b85-9dff-9215019cdad5.png)
![image](https://user-images.githubusercontent.com/107001681/220340602-954dc867-fb56-424b-b840-14fd50aadb10.png)

**Reply by reginaldStjohn** [reply]
When I got my system to work I individually installed some items as I saw their errors appear .  this may not be your problem though.

sudo apt update
pip3 install PyQt5
pip3 install qasync
sudo apt-get install packer

**Comment by reginaldStjohn** [comment]
Yes, I figured that I could manually update. Just wanted to let the "team" know and maybe update instructions or the virtual machine.
